pkg/exposer/generic_restore.go

```diff
@@ -370,6 +370,7 @@ func (e *genericRestoreExposer) createRestorePod(ctx context.Context, ownerObjec
 					VolumeMounts:  volumeMounts,
 					VolumeDevices: volumeDevices,
 					Env:           podInfo.env,
+					EnvFrom:       podInfo.envFrom,
```

Comment:
Please do the same for `createBackupPod` (exposer/csi_snapshot.go)

Reply:
Done. :man_facepalming: